(* This file was auto-generated based on "parser.messages". *)

(* Please note that the function [message] can raise [Not_found]. *)

let message s =
  match s with
  | 0 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 1 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 5 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 275 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 278 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 285 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 281 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 13 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 15 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 16 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 17 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 19 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 20 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 21 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 22 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 23 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 45 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 51 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 50 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 81 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 105 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 53 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 77 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 86 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 83 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 132 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 46 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 156 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 48 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 89 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 90 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 91 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 92 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 94 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 95 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 99 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 42 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 31 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 140 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 141 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 147 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 148 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 144 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 150 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 145 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 142 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 139 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 32 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 125 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 54 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 34 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 135 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 136 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 137 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 134 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 36 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 111 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 112 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 113 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 114 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 110 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 119 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 120 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 121 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 122 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 123 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 117 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 61 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 62 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 127 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 63 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 64 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 68 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 69 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 65 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 66 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 41 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 157 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 158 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 177 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 172 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 163 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 173 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 160 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 166 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 178 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 180 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 181 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 183 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 182 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 184 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 187 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 188 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 190 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 191 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 189 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 161 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 169 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 200 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 196 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 197 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 198 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 204 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 207 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 208 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 209 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 210 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 211 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 212 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 213 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 237 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 230 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 231 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 235 -> "<YOUR SYNTAX ERROR MESSAGE HERE>\n"
  | 224 -> "expected a new struct item, or another declaration or scope use\n"
  | 227 -> "expected a new struct item, or another declaration or scope use\n"
  | 240 -> "expected the name of the scope you are declaring\n"
  | 241 -> "expected a colon followed by the list of context items of this scope\n"
  | 242 -> "expected a context item introduced by \"context\"\n"
  | 243 -> "expected the name of this new context item\n"
  | 244 -> "expected the kind of this context item: is it a condition, a sub-scope or a data?\n"
  | 245 -> "expected the name of the subscope for this context item\n"
  | 252 -> "expected the next context item, or another declaration or scope use\n"
  | 247 -> "expected the type of this context item\n"
  | 248 -> "expected the next context item or a dependency declaration for this item\n"
  | 250 -> "expected the next context item or a dependency declaration for this item\n"
  | 255 -> "expected the name of your enum\n"
  | 256 -> "expected a colon\n"
  | 257 -> "expected an enum case\n"
  | 258 -> "expected the name of an enum case \n"
  | 259 -> "expected a payload for your enum case, or another case or declaration \n"
  | 260 -> "expected a content type\n"
  | 265 -> "expected another enum case, or a new declaration or scope use\n"
  | 11 -> "expected a declaration or a scope use\n"
  | 12 -> "expected a declaration or a scope use\n"
  | 271 ->
      "should not happen, please file an issue at https://github.com/CatalaLang/catala/issues\n"
  | _ -> raise Not_found
